harfbuzz_overflow_resolution.md

```diff
@@ -0,0 +1,157 @@
+# Introduction
+
+Several tables in the opentype format are formed internally by a graph of subtables. Parent node's
+reference their children through the use of positive offsets, which are typically 16 bits wide.
```

Comment:
perhaps worth explicitly noting that due to positive offsets cycles cannot occur? unless there is a "jump backward" mechanism, in which case note that cycles *can* occur